code/modules/reagents/chemistry/holder.dm

```diff
@@ -1,665 +0,0 @@
-//This file was auto-corrected by findeclaration.exe on 25.5.2012 20:42:32
```

Comment:
Why didn't you move all these lines to a new file just to change the file's name? It makes it very hard to spot the changes you've made in these lines.